dali/operators/image/crop/crop_attr.h

```diff
@@ -58,6 +58,7 @@ class CropAttr {
   std::vector<float> crop_z_norm_;
   std::vector<CropWindowGenerator> crop_window_generators_;
   bool is_whole_image_ = false;
+  std::function<float(float)> round_fn_;
```

Comment:
```suggestion
  std::function<int(float)> round_fn_;
```
Don't we want to return integers?
Also, are we good with just `float` (and not double)?